app/src/main/java/co/co/nimblehq/showcases/poly/place/ui/screens/home/NearbyRestaurantsScreen.kt

```diff
@@ -1,12 +1,17 @@
 package co.co.nimblehq.showcases.poly.place.ui.screens.home
```

Comment:
Could you please change the package to:
```suggestion
package co.co.nimblehq.showcases.poly.place.ui.screens.restaurant.nearby
```

So that I will create another directory for restaurant detail like this:
```
package co.co.nimblehq.showcases.poly.place.ui.screens.restaurant.details
```